examples/controlnet/train.sh

```diff
@@ -13,14 +32,12 @@ else
   batch_size=5
 fi
 
-moreh-switch-model -M 1 
-
 # Run training script
 echo "# ========================================================= #"
 echo "training ${model_name}.."
 conda run -n ${env_name} python3 train_controlnet.py \
-    --pretrained_model_name_or_path ${model_name}  \
-    --controlnet_model_name_or_path lllyasviel/sd-controlnet-hed \
+    --pretrained_model_name_or_path runwayml/stable-diffusion-v1-5  \
```

Comment:
Is this one the only option for pretrained model? Wondering why this specific model name is hardcoded here.

Reply:
Controlnet task uses two types of models: stable diffusion and contronet at the same time. The stable diffusion models are trained specifically for the text_to_image task. Therefore, in the controlnet task, the script is modified to accommodate the different controlnet models.